spec/models/tag_spec.rb

```diff
@@ -11,6 +11,56 @@
     User.current_user = nil
   end
 
+  context "checking count caching" do
+
+    before(:each) do
+      # Set the minimail amount of time a tag can be cached for.
+      ArchiveConfig.TAGGINGS_COUNT_MIN_TIME = 1
+      # Set so that we need few uses of a tag to start caching it.
+      ArchiveConfig.TAGGINGS_COUNT_CACHE_DIVISOR = 2
+      # Set the minimum number of uses needed for caching is possible
+      ArchiveConfig.TAGGINGS_COUNT_MIN_CACHE_COUNT = 3
+      @fandom_tag = FactoryGirl.create(:fandom)
+    end
+
+    it "A small tag should be uneffected by caching" do
```

Comment:
"unaffected". Also, the reason Rspec methods start with "it" is so it can produce a nice output with complete sentences like "it should not cache a small tag"